### PR TITLE
docs(element-templates): storage location of synced templates

### PR DESCRIPTION
**Pull request by MaxTru**
__Which issue does this PR address?__

Closes #

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by philippfromme** [comment]
Are we sure we want to document this publically? This is an implementation detail that could change in the future. I'd also not want our users to mess with the `config.json`.  @azeghers @barmac @nikku @pinussilvestrus 👨 👨 👨👨

**Comment by MaxTru** [comment]
I believe we have the use-case: "I want to locally delete the element templates, which I once synced using Cawemo". Currently the only way to do that, is to delete the templates from Cawemo. In my use case, this was not an option since they were not my own (they were just shared with me).

Hence the documentation of this is needed.

**Comment by philippfromme** [comment]
Okay, so this is more of a workaround since there is no functionality in the Camunda Modeler to remove synchronized templates?

File: docs/element-templates/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Element templates allow you to create pre-defined configurations for BPMN elemen
 
 Element templates are defined as [JSON files](#defining-templates). There are two ways to integrate them into the Camunda Modeler:
 
-1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo.
+1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo. Locally the sycned templates will be stored in the `config.json` file in your [`{USER_DATA_DIRECTORY}`](https://github.com/camunda/camunda-modeler/tree/master/docs/search-paths#user-data-directory). Note that the [Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) will keep Cawemo and your local Camunda Modeler installation in sync (e.g., deleting a template in Cawemo will delete it locally as well).
```
Comment (barmac) [inline docs/element-templates/README.md:22]: > sycned

Typo.

Comment (MaxTru) [reply docs/element-templates/README.md:22]: ty :+1:

Comment (barmac) [reply docs/element-templates/README.md]: BTW I'm not sure if this is detailed enough to actually remove the templates manually.

**Comment by barmac** [comment]
I think if we are going to add this workaround, let's also be clear that changing config is a power user action.

File: docs/element-templates/README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Element templates allow you to create pre-defined configurations for BPMN elemen
...
 
-1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo.
+1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo. Note that the [Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) will keep Cawemo and your local Camunda Modeler installation in sync (e.g., deleting a template in Cawemo will delete it locally as well). Locally the synced templates will be stored in the `config.json` file in your [`{USER_DATA_DIRECTORY}`](https://github.com/camunda/camunda-modeler/tree/master/docs/search-paths#user-data-directory). Making manual changes to the `config.json` is not recommended and should be a "power-user" action only.
```
Comment (philippfromme) [inline docs/element-templates/README.md:22]:
```suggestion
1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo. Note that the [Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) will keep Cawemo and your local Camunda Modeler installation in sync (e.g., deleting a template in Cawemo will delete it locally as well). Locally the synced templates will be stored in the `config.json` file in your [`{USER_DATA_DIRECTORY}`](https://github.com/camunda/camunda-modeler/tree/master/docs/search-paths#user-data-directory). You should not manually change the `config.json` unless you know what you're doing.
```

Comment (philippfromme) [reply docs/element-templates/README.md:22]: I'd not use the term _power-user_ as I'd argue that manually changing internals falls outside of the user category.

**Pull request merged**